sdk/core/Azure.Core/tests/ModelSerializationTests/IUtf8JsonSerializable.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+
+namespace Azure.Core.Tests.ModelSerializationTests
+{
+    internal interface IUtf8JsonSerializable
+    {
+        void Write(Utf8JsonWriter writer, SerializableOptions options);
```

Comment:
Since `IUtf8JsonSerializable` is internal, we will just update the signature of the existing `IUtf8JsonSerializable`?